src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/illegalimport/InputIllegalImportDefault.java

```diff
@@ -97,5 +97,6 @@ public void render() {}
      * @throws TestClass7 when broken
      * @deprecated in 1 for removal in 2. Use {@link TestClass8}
      */
+    @Deprecated
```

Comment:
Why do we need this for a non-compilable class that says it doesn't support 10+ ?

Reply:
Come JDK classes were removed (actually, they are hidden) in Java9. This makes it non-compilable fog Java9+.
But this test input file can be compilable. I'll try to find packages available for both Java8 and Java11.

Reply:
That doesn't explain why the `@Deprecated` was added.

Reply:
Yes, javac11 complains about a method with the Javadoc tag `@deprecated`, but without the `@Deprecated` annotation.
```java
public class TestClass {
  /**
   * @deprecated use bar instead
   */
  public void foo() {
  }
}
```
```bash
$ javac --version TestClass.java 
javac 11.0.10
TestClass.java:5: warning: [dep-ann] deprecated item is not annotated with @Deprecated
  public void foo() {
```